backend/web/ui/static/js/app.js

```diff
@@ -63,3 +62,123 @@ function user_login() {
         dataType: "json",
     });
 }
+
+function add_new_todo(todo_status_id) {
+    let form_id = "#form_" + todo_status_id
+    $(form_id).trigger("reset");
+    var modal_id = "#modal_" + todo_status_id
+    $(form_id).removeClass('was-validated')
+    $(modal_id + " #submit_values").attr("onclick", "insert_new_todo(\"" + form_id + "\"," + "\"" + todo_status_id + "\")")
+    $(modal_id + " #add_todo_modal").attr("onclick", "close_modal_window(\"" + modal_id + "\")")
+    $(modal_id).modal('show')
+}
+
+function close_modal_window(modal_id){
+    $(modal_id).modal('hide');
+}
+
+function insert_new_todo(form_id, todo_status_id) {
+    let data = {
+        "title": $(form_id + " input[name=title]").val(),
+        "description": $(form_id + " #" + todo_status_id + "_description").val(),
+        "priority": $(form_id + " #" + todo_status_id + "_priority").val(),
+        "image": $(form_id + " input[name=image]").val(),
+        "status": todo_status_id
+    }
+    $.ajax({
+        type: "POST",
+        url: "/todo",
+        data: JSON.stringify(data),
+        success: function (response, textStatus) {
+            if (textStatus == "success") {
+                window.location = '/';
+            } else {
+                let response_msg = jQuery.parseJSON(response.responseText)
+                error_alert(response_msg['msg'], form_id)
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/IrinaLokhvytska/owasp/pullRequest?prid=8125955)
```suggestion
                error_alert(response_msg["msg"], form_id)
```